### DISCUSSION TITLE
How can I `display` variable like standard gdb does?

### DISCUSSION THREAD

**Discussion by yayuwang0624** [hugsy/gef#952] [Q&A] [answered]
I tried `display i` but it does not show the value of i in the following execution

**Comment by hugsy** [comment]
Use `print`

**Reply by yayuwang0624** [reply]
but it is a one-time `print` not a persistent `display` right?

**Reply by hugsy** [reply]
What do you mean by "persistent"? Do you want to make it part of the context in GEF?

**Reply by yayuwang0624** [reply]
yes.

like in the original gdb once you `display` a variable, it will be always printed after you execute `next`, `continue` etc.

can I also do this with gef?

**Reply by hugsy** [reply, marked as answer]
there are several ways but the easiest would be to use the `memory watch` command, or you can also create your own context pane.
both ways are documented, that should help you to start.
